toolbox/cluster.py

```diff
@@ -6,7 +6,7 @@ class Cluster:
     Commands relating to cluster scaling, upgrading and environment capture
     """
     @staticmethod
-    def set_scale(instance_type, scale, force=False):
+    def set_scale(instance_type, scale, base_machineset="", force=False):
```

Comment:
Might want to fix the docstring - it doesn't explain anymore what happens when a user specifies a `base_machineset`

Reply:
why not? there is this below (I'll removed the `s` of `defaults`)
```
             base_machineset: Name of a machineset to use to derive the new one. Defaults: pickup the first machineset found in `oc get machinesets -n openshift-machine-api`.
```

Reply:
The entire description about the parameters docs doesn't take this parameter into account, and it's unclear what would happen when it's specified

Reply:
updated